docs/src/administration/backup-and-restore.md

```diff
@@ -72,7 +72,7 @@ $ platform backup:create --live
 
 ### Automated backups
 
-Backups aren't triggered automatically on Platform.sh Professional.
+Backups aren't triggered automatically on Platform.sh Professional. For Enterprise and Elite [see Backups](../../dedicated/overview/backups.md).
```

Comment:
```suggestion
Backups aren't triggered automatically on Platform.sh Professional plans.
For Enterprise and Elite plans, see [backups](../../dedicated/overview/backups.md).
```

Reply:
Also, isn't Dedicated a separate question from the plan itself? I think it's possible to have Enterprise support without a Dedicated cluster. Or no?